openedx/core/djangoapps/xblock/tests/test_utils.py

```diff
@@ -0,0 +1,108 @@
+"""
+Tests for xblock utils.
+"""
+import datetime
+
+import pytest
+from django.test import override_settings
+from freezegun import freeze_time
+
+from openedx.core.djangoapps.xblock.utils import (
+    get_secure_token_for_xblock_handler,
+    validate_secure_token_for_xblock_handler
+)
+
+REFERENCE_PARAMS = {
+    "generation_user_id": "12344",
+    "validation_user_id": "12344",
+    "generation_block_key_str": "some key",
+    "validation_block_key_str": "some key",
+    "generation_secret_key": "baseline_key",
+    "validation_secret_key": "baseline_key",
+    "generation_xblock_handler_token_keys": None,
+    "validation_xblock_handler_token_keys": None,
+    "reference_time": datetime.datetime(2021, 1, 28, 13, 26, 38, 787309),
```

Comment:
Probably want to add a time zone here.

Reply:
Likely also want to mention how this timestamp was chosen, or document the distance from the most recent TIME_PERIOD boundary.

Reply:
(We can make these changes on our PR, actually.)